cmd/documentation/exchanges_templates/hitbtc.tmpl

```diff
@@ -93,12 +93,24 @@ if err != nil {
 }
 ```
 
-### How to do Websocket public/private calls
+### Subscriptions
 
-```go
-	// Exchanges will be abstracted out in further updates and examples will be
-	// supplied then
-```
+Subscriptions are for [v2 api](https://hitbtc-com.github.io/hitbtc-api/#socket-api-reference)
+
+All subscriptions are for spot.
+
+Default Public Subscriptions:
```

Comment:
NOTE 4 ME: Might be able to make this entire change dynamic across all exchanges in #1619 reduce implementation toil.